docs/slurm/sbatch.md

```diff
@@ -898,6 +906,36 @@ This command indicates that `job2.sbatch` will be put in the queue after the job
 
 For more information on job dependencies, please refer to the [Slurm documentation](https://slurm.schedmd.com/sbatch.html){:target='_blank'}.
 
+### Cron-like jobs - Midway2 - Legacy 
+
+Cron-like jobs are jobs submitted to the queue with a specified schedule. These jobs persist until they are canceled or encounter an error. The Midway2 cluster has a dedicated partition, `cron`, for running cron-like jobs. Please contact our [Help Desk](https://rcc.uchicago.edu/support-and-services/consulting-and-technical-support){:target='_blank'} to request submitting Cron-like jobs. These jobs are subject to scheduling limits and will be monitored. We strongly recommend using `dependency jobs` rather than `cron jobs` since cron-like jobs is a legacy service and we are moving away from it. 
```

Comment:
"cron-like jobs considered as a legacy"?

Reply:
I made changes to the text, not as a legacy, but rather limited, or reluctant, support.